src/server/config.rs

```diff
@@ -16,11 +16,13 @@ use super::Result;
 
 const DEFAULT_CLUSTER_ID: u64 = 0;
 pub const DEFAULT_LISTENING_ADDR: &'static str = "127.0.0.1:20160";
+pub const DEFAULT_LOCAL_STORE_PATH: &'static str = "";
 const DEFAULT_ADVERTISE_LISTENING_ADDR: &'static str = "";
 const DEFAULT_NOTIFY_CAPACITY: usize = 4096;
 const DEFAULT_MESSAGES_PER_TICK: usize = 256;
 const DEFAULT_SEND_BUFFER_SIZE: usize = 128 * 1024;
 const DEFAULT_RECV_BUFFER_SIZE: usize = 128 * 1024;
+const DEFAULT_STORAGE_SCHED_CONCURRENCY: usize = 1024;
```

Comment:
Is `CONCURRENCY` a suitable name here? Maybe `LATCH_SIZE`? `CONCURRENCY` sounds like we can execute 1024 requests at a same time.